src/program-proofs/chapter08/insertion_sort.ml

```diff
@@ -5,20 +5,24 @@ let[@pure] rec is_sorted (xs: int list): bool =
   | x :: xs' -> (
     match xs' with
     | [] -> true
-    | x' :: xs'' -> x <= x' && is_sorted xs'
+    (* Note: && is not yet supported *)
```

Comment:
This should be easy to add, if you would like to try it. Same for || below.

Reply:
Sure!

Reply:
I have added it:

<details><summary>Example</summary>

```ocaml
let[@pure] hello (n: int) : bool =
  (n = 1 || n = 2) && n > 0
```

```why3
module M
  use int.Int
  use bool.Bool
  use list.List
  
  function hello (n: Int.int) : Bool.bool =
    let v2 =
      let v0 = Int.( = ) n 1 in
      let v1 = Int.( = ) n 2 in
      Bool.orb v0 v1 in
    let v3 = Int.( > ) n 0 in
    Bool.andb v2 v3
  
  goal goal1:
    forall n : Int.int, res : Bool.bool, v4 : Bool.bool, v5 : Bool.bool, v6 : 
          Bool.bool, v7 : Bool.bool.
      (((((v5 = (n = 1)) /\ (v4 = (n = 2))) /\ (v7 = (Bool.orb v5 v4)))
          /\
          (v6 = (Int.( > ) n 0)))
         /\
         (res = (Bool.andb v7 v6)))
        ->
        false
end
```
</details>

Reply:
Great, thanks!